Update EOS examples for the new api on the Jungle 2.0 testnet (#2)
oraclize/eos-examples: url-requests-computation/README.md

# Oraclize's URL Requests Computation Example 

This repo is to demonstrate how you would work with the Oraclize **computation** datasource.

> The `urlrequests` example allows you to *retrive the response of several URL requests*.

Reviewer: ```suggestion
The `urlrequests.cpp` example allows you to retrieve the response of several URL requests.
```

## Notes

> Oraclize replies to your `oraclize_query` with the *respose body result* calling your `callback(...)`. 

Reviewer: ```suggestion
Oraclize replies to your `oraclize_query` by calling your `callback(...)` with the *respose body result*. 
```

> You can search your transaction ID in one of the following links to verify it:

Reviewer: ```suggestion
You can search for your transaction ID in one of the following links to verify it:
```

> * :mag_right::ledger: [Blocks.io](https://jungle.bloks.io/): a block explorer for the Jungle 2.0 testnet.

Reviewer: ```suggestion
* :mag_right::ledger: [Blocks.io](https://jungle.bloks.io/): A block explorer for the Jungle 2.0 testnet.
```

> * :palm_tree::lion::palm_tree: [Jungle 2.0](https://monitor.jungletestnet.io/#home): a transaction explorer is available selecting *Get TX* on the Jungle 2.0 testnet website.

Reviewer: ```suggestion
* :palm_tree::lion::palm_tree: [Jungle 2.0](https://monitor.jungletestnet.io/#home): A transaction explorer is available by selecting *Get TX* on the Jungle 2.0 testnet website.
```
